tests/golden_tests.rs

```diff
@@ -169,3 +170,23 @@ fn encode_pattern_match() {
     Ok(book.to_string())
   })
 }
+
+#[test]
+#[ignore = "To not delay golden tests execution"]
+fn hangs() {
+  let expected_normalization_time = 1;
```

Comment:
Is 1 second enough? I haven't checked how long a basic program takes to compile and run